src/main/java/org/candlepin/model/ProductCurator.java

```diff
@@ -122,7 +122,7 @@ public Set<Product> getPoolProvidedProductsCached(String poolId) {
     public Set<String> getPoolProvidedProductUuids(String poolId) {
         TypedQuery<String> query = getEntityManager().createQuery(
             "SELECT product.uuid FROM Pool p INNER JOIN p.product.providedProducts product " +
-            "where p.id = :poolid",
+                "where p.id = :poolid",
```

Comment:
Line continuation.

Reply:
In this case, the indentation is a requirement of the query itself being too long because everything is being chained on one line. So, we could do what we've done in other places, define the query statement above, and then use the per-line chaining for the rest:

```
        String jpql = "SELECT product.uuid " +
            "FROM Pool p INNER JOIN p.product.providedProducts product " +
            "WHERE p.id = :poolid";

        TypedQuery<String> query = this.getEntityManager()
            .createQuery(jpql, String.class)
            .setParameter("poolid", poolId);
            
        return new HashSet<>(query.getResultList());
```

Reply:
Alright, so we meant the same thing. Done.